pom.xml

```diff
@@ -39,7 +39,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>2.73.3</jenkins.version>
```

Comment:
Please don't change this as it represents the first Jenkins version we support.

Reply:
I needed to change this version because some jenkins plugins weren't working on my machine, like git plugin. I'll give it a try again and keep you posted.

Reply:
I think what Vitor was saying is that, yes, go ahead and change this to whatever you want, but don't commit it as it's related to the version we support.

Reply:
Yes, that's what i got but you don't want to change it every time you have to develop some new feature. It seems that there are some incompatibility between this jenkins version and some plugin versions we are using in our development. Well, i'm just going to revert this change because we don't need to wait to handle those incompatibilities do make the plugin available for customers. Probably we'll need to address it soon.